app/org/sagebionetworks/bridge/services/UserAdminService.java

```diff
@@ -163,9 +163,25 @@ public UserSession createUser(SignUp signUp, Study study, SubpopulationGuid subp
     }
 
     /**
+     * Sign out a user's session.
+     *
+     * @param study
+     *              study of target user
+     * @param email
+     *              target user
+     */
+    public void invalidateUserSession(Study study, String email) {
+        checkNotNull(study);
```

Comment:
I don't think you need to retrieve the user's account again, as in the controller when you retrieve the session, it has a user, and the user has the same ID you're retrieving to invalidate the session. You might change this signature to take the ID directly.